Backpack/Button/Classes/BPKButton.m

```diff
@@ -42,8 +42,13 @@ @interface BPKButton ()
 @property(nonatomic, class, readonly) UIColor *highlightedOutline;
 @property(nonatomic, class, readonly) UIColor *highlightedBlue;
 @property(nonatomic, class, readonly) UIColor *highlightedRed;
+@property(nonatomic, class, readonly) UIColor *disabledLinkColor;
```

Comment:
Why these colors are `class` properties? are we going to use them outside a particular `BPKButton` instance?